### DISCUSSION TITLE
Add option to suspend window, applicable when in dev containers

### DISCUSSION THREAD

**Discussion by naturedamends** [devcontainers/community#55] [Ideas]
I'm using mac, and running the virtual machine in the background is expensive. I would like to suspend the window such that I can keep the window open while not paying the expenses of keeping all the secondary applications running. 

I"m happy to write a PR or help in someway. But the extend of the upgrade might be large if one takes into account the window and the dev containers feature. idk

I guess this could be implemented with a simple call to docker stop [dev container] since the window already suspends 
But would be nice to keep the window intractable and re connect on interaction

```
> Developer: suspend window
> Dev containers:  suspend  dev container
```

Thoughts?

Edit:

In my case it may be better to say: suspend extensions, since it seems to be the ping between host and remote extension that is triggering massive cpu spikes.

It's also my fault since I have a dev container with a ton of extensions, and like to keep a ton of windows open

**Comment by bamurtaugh** [comment]
Thank you for opening @naturedamends!

@alexander-smolyakov - I'd be interested in your thoughts on this one!
